config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  # get 'menu/index'
+
+  resources :menu_items
+
+  root 'menu#index'
```

Comment:
please refactor this to the new syntax

e.g.
```
root controller: :menu_items, action: :index
```

Reply:
Syntax updated.